storage/innobase/xtrabackup/src/CMakeLists.txt

```diff
@@ -203,6 +204,7 @@ TARGET_LINK_LIBRARIES(xbstream
   mysys
   crc
   ${LZ4_LIBRARY}
+  mysqlclient
```

Comment:
why do we have to link mysqclient to xbstream?

Reply:
ut0ut.h requires mysqld_Error.h which only gets generates by mysqlclient target:

```
In file included from /work/pxb/src/8.0/storage/innobase/include/ut0cpu_cache.h:33,
                 from /work/pxb/src/8.0/storage/innobase/include/ut0counter.h:42,
                 from /work/pxb/src/8.0/storage/innobase/include/sync0sync.h:45,
                 from /work/pxb/src/8.0/storage/innobase/include/sync0types.h:39,
                 from /work/pxb/src/8.0/storage/innobase/include/univ.i:546,
                 from /work/pxb/src/8.0/storage/innobase/include/fsp0fsp.h:36,
                 from /work/pxb/src/8.0/storage/innobase/xtrabackup/src/file_utils.cc:11:
/work/pxb/src/8.0/storage/innobase/include/ut0ut.h:72:10: fatal error: mysqld_error.h: No such file or directory
   72 | #include "mysqld_error.h"
      |          ^~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [storage/innobase/xtrabackup/src/CMakeFiles/xbstream.dir/build.make:167: storage/innobase/xtrabackup/src/CMakeFiles/xbstream.dir/file_utils.cc.o] Error 1
make[2]: *** Waiting for unfinished jobs....

```

Reply:
Please avoid this at all cost. I think, we can have InnoDB Page parser class. it is OK to duplicate for parsing purpose.

Reply:
fixed